src/main/groovy/edu/oregonstate/mist/locations/core/ExtensionLocation.groovy

```diff
@@ -0,0 +1,34 @@
+package edu.oregonstate.mist.locations.core
+
+class ExtensionLocation {
+    private static final String GEO_LOCATION_REGEX = "-?\\d+(\\.\\d+)?"
```

Comment:
Extension and Dining are using different string formats to specify their lat / long. Technically one more complex regex could handle both types of strings, but I want to keep the regex small and simple.